core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -18,8 +18,11 @@ final case class Kleisli[F[_], A, B](run: A => F[B]) { self =>
   def map[C](f: B => C)(implicit F: Functor[F]): Kleisli[F, A, C] =
     Kleisli(a => F.map(run(a))(f))
 
-  def mapF[N[_], C](f: F[B] => N[C]): Kleisli[N, A, C] =
-    Kleisli(run andThen f)
```

Comment:
Maybe we can keep this as a deprecated alias? :)

Reply:
I didn't mean to remove it-- I had added `mapF` to every transformer but wound up splitting off `mapK` additions to a separate PR. This just an accidental carry over from the split!